### ISSUE TITLE
Setup taskwiki with Packer and lua

### ISSUE THREAD

**Issue by mislavzlatar** [tools-life/taskwiki#414]
Hi I love your plugin, but when I migrated my neovim's `init.vim` to `init.lua` I see `taskwiki` is not being loaded. For example if I type `:taskwiki` I get `not an editor command`.

I have this in my `plugins.lua` for install it with `Packer`:

```lua
use {
  "tbabej/taskwiki"
}
```

and I see plugin got installed in `~/.local/share/nvim/site/pack/packer/start/taskwiki`

How can I debug this issue? Or is it even possible for `taskwiki` to worki with `packer` and `lua`?

Thanks in advance,

Mislav

**Comment by ibrahimkettaneh** [comment]
> Hi I love your plugin, but when I migrated my neovim's `init.vim` to `init.lua` I see `taskwiki` is not being loaded. For example if I type `:taskwiki` I get `not an editor command`.
> 
> I have this in my `plugins.lua` for install it with `Packer`:
> 
> ```lua
> use {
>   "tbabej/taskwiki"
> }
> ```
> 
> and I see plugin got installed in `~/.local/share/nvim/site/pack/packer/start/taskwiki`
> 
> How can I debug this issue? Or is it even possible for `taskwiki` to worki with `packer` and `lua`?
> 
> Thanks in advance,
> 
> Mislav

You can ue packer with taskwiki, it's just that like you, I have found exactly 0 examples of doing this
Similarly to vimwiki, the taskwiki configuration only takes effect after doing a reboot, at least for me
Here is my current lua configuration, which means taskwiki can be configureed with packer. I hope it helps
```lua
  {
    'tools-life/taskwiki',
    config = function()
      taskwiki_settings = {
        taskwiki_taskrc_location = 'path/to/config',
        taskwiki_data_location = 'path/to/taskwarrior/directory/',
      }
    end,
  },
````

I would greatly appreciate any examples of using this taskwiki setting please
`taskwiki_extra_warriors = {'H': {'data_location': '/home/tbabej/.habit/', 'taskrc_location': '/home/tbabej/.habitrc'}`

**Comment by mislavzlatar** [comment]
Hey @ibrahimkettaneh  thanks for reply..


I tried your config and rebooted pc but still it doesn't work..

Little bit about my environment:

```
PRETTY_NAME="Ubuntu 22.04.1 LTS"

5.15.0-50-generic

❯ n --version 
NVIM v0.7.2

:checkhealth


## Python 3 provider (optional)
  - INFO: `g:python3_host_prog` is not set.  Searching for python3 in the environment.
  - INFO: Multiple python3 executables found.  Set `g:python3_host_prog` to avoid surprises.
  - INFO: Executable: /usr/bin/python3
  - INFO: Other python executable: /bin/python3
  - INFO: Python version: 3.10.6
  - INFO: pynvim version: 0.4.3
  - OK: Latest pynvim is installed.

## Python virtualenv
  - OK: no $VIRTUAL_ENV

task --version
2.6.1
```

**Comment by mislavzlatar** [comment]
Found solution on this [link](https://www.reddit.com/r/taskwarrior/comments/x3dr9h/taskwiki_workingbut_taskwiki_tasks_not_migrating/)

tldr you have to install `packaging` `python3` package

```
pip3 install packaging
```

All working fine now, closing issue   :)

**Comment by mislavzlatar** [comment]
https://github.com/tools-life/taskwiki/issues/395
